src/foam/u2/view/OverlayActionListView.js

```diff
@@ -190,20 +190,20 @@ foam.CLASS({
       (this.firstEl_ && ! this.isMouseClick) && this.firstEl_.focus();
     },
 
-    function onKeyDown(e) {
+    async function onKeyDown(e) {
       var isTabPressed = (e.key === 'Tab' || e.keyCode === 9);
 
       if ( ! isTabPressed ) {
         return;
       }
 
       if ( e.shiftKey ) {
-        if ( this.document.activeElement === this.firstEl_.el() ) {
+        if ( this.document.activeElement === await this.firstEl_.el() ) {
```

Comment:
You could also just call el_() instead. Since this is a DOM listener, it's safe to assume the element is currently loaded and then call the sync version.